### PR TITLE
Blog template text updates

### PR DESCRIPTION
**Pull request by ericlindley-g**

### CODE REVIEW

**Comment by ericlindley-g** [comment]
I think because the content is being tested here: https://github.com/ampproject/ampstart/blob/6dc33a89c1a74045a30808eb5cc5ae6c0679ee60/templates/test-all.amp.json  ?

**Comment by camelburrito** [comment]
The JSON is invalid , someplace you broke the JSON.

File: templates/blog-recipe.amp.json [hunk 1]
```diff
@@ -139,7 +113,6 @@
         {"facebook": "#"},
         {"instagram": "#"},
         {"pinterest": "#"},
```
Comment (camelburrito) [inline templates/blog-recipe.amp.json]: remove this comma and that will fix it

**Comment by ericlindley-g** [comment]
Removed —those commas get me every time...

**Pull request merged**